src/java.base/share/classes/sun/security/util/Cache.java

```diff
@@ -360,26 +392,64 @@ public synchronized void clear() {
             for (CacheEntry<K,V> entry : cacheMap.values()) {
                 entry.invalidate();
             }
-            while (queue.poll() != null) {
-                // empty
-            }
+            while (queue.poll() != null);
```

Comment:
Can we keep the original code? IntelliJ complains about both versions, but the comment makes it more clear that the empty loop is intentional.